### PR TITLE
Added Bolsa Supply Midway60 Keyboard

### PR DESCRIPTION
**Pull request by mrnoisytiger**
## Description

Added Bolsa Supply Midway60 Keyboard

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by mrnoisytiger** [comment]
New pull request for previously reviewed https://github.com/qmk/qmk_firmware/pull/12439 to fix submodules problem.

File: keyboards/fjlabs/midway60/rules.mk [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+COMMAND_ENABLE = yes        # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = yes           # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+BLUETOOTH_ENABLE = no       # Enable Bluetooth
+AUDIO_ENABLE = no           # Audio output
+KEYBOARD_LOCK_ENABLE = yes
```
Comment (drashna) [inline keyboards/fjlabs/midway60/rules.mk:26]:
```suggestion
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Comment by mrnoisytiger** [comment]
@drashna Changed!

File: keyboards/fjlabs/midway60/rules.mk [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = yes           # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+BLUETOOTH_ENABLE = no       # Enable Bluetooth
+AUDIO_ENABLE = no           # Audio output
+
+LAYOUTS = all
```
Comment (drashna) [inline keyboards/fjlabs/midway60/rules.mk:27]:
```suggestion
```

File: keyboards/fjlabs/midway60/readme.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+The PCB will feature:
+* QMK & VIA compatibility
+* Some cool bolsa branding
+ 
+---
+
+* Keyboard Maintainer: FJLabs
+* Hardware Supported: Midway60
+
+Make example for this keyboard (after setting up your build environment):
```
Comment (drashna) [inline keyboards/fjlabs/midway60/readme.md:14]: Could you add instructions on how to jump to the bootloader?

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/fjlabs/midway60/readme.md (hunk 3 above)
Comment (mrnoisytiger) [reply keyboards/fjlabs/midway60/readme.md:14]: Added.

**Review by drashna — APPROVED** [review]

**Review by fauxpark — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Pull request merged**